tests/gold_tests/thread_config/check_threads.py

```diff
@@ -23,12 +23,11 @@
 
 def count_threads(ts_path, etnet_threads, accept_threads, task_threads, aio_threads):
 
-    for pid in psutil.pids():
+    for p in psutil.process_iter():
```

Comment:
Looks good.

As a tweak, reading the docs, it looks like the preferable way to do this is by passing in the subset of process attributes that are desired. In this case, perhaps:

```
for p in psutil.process_iter(['name', 'cwd', 'threads']):
   ...
   p.info['name'] == '[TS_MAIN]' and p.info['cwd'] == ts_path:
        ...
```

Reply:
done

Reply:
Great. I'm glad you used p.name rather than p.info['name'] like I suggested. That's much nicer.